nouislider/nouislider.d.ts

```diff
@@ -16,23 +16,23 @@ declare namespace noUiSlider {
          */
         start: number | number[] | number[][];
         /**
-        * The connect setting can be used to control the bar between the handles,
-        * or the edges of the slider. Use "lower" to connect to the lower side,
-        * or "upper" to connect to the upper side. Setting true sets the bar between the handles.
-        */
+         * The connect setting can be used to control the bar between the handles,
+         * or the edges of the slider. Use "lower" to connect to the lower side,
+         * or "upper" to connect to the upper side. Setting true sets the bar between the handles.
+         */
         range: Object;
         /**
          * noUiSlider offers several ways to handle user interaction.
          * The range can be set to drag, and handles can move to taps.
          * All these effects are optional, and can be enable by adding their keyword to the behaviour option.
          * This option accepts a "-" separated list of "drag", "tap", "fixed", "snap" or "none".
          */
-        connect?: string | boolean;
+        connect?: boolean | boolean[];
```

Comment:
I may misunderstand something but should this be just `boolean[]` as in this doc

> Removed: connect: 'lower' and connect: 'upper'. These settings are replaced by connect: [true, false]; 

Also should the comment be updated?